src/crossroads/main.py

```diff
@@ -185,8 +191,80 @@ def validate_json(json_content):
     return True
 
 
-def data_which_overlap():
-    pass
+def data_which_overlap(first_timestamp_list, second_timestamp_list):
```

Comment:
Names of these arguments are confusing since these lists will not contain only timestamps, but whole data.